daemon/monitor.go

```diff
@@ -128,6 +128,11 @@ func (daemon *Daemon) ProcessEvent(id string, e libcontainerd.EventType, ei libc
 			// remove the exec command from the container's store only and not the
 			// daemon's store so that the exec command can be inspected.
 			c.ExecCommands.Delete(execConfig.ID, execConfig.Pid)
+			attributes := map[string]string{
+				"execId":   execConfig.ID,
```

Comment:
Generally, I think this PR looks good; I want to double check on this one with other maintainers. I think the other `exec` requests;

- have `exec_foo:<command and arguments>` as event (this was actually a mistake, but we had to keep it for backward-compatibility, and still have to work on improving that)
- do _not_ have an `execId` property; I think it's useful, but wondering if we should see this as a separate feature, and also consider adding the attribute to other `exec_xx` events (so that events can be related to each other)

FWIW, there's some weird things still with the events endpoint (e.g. "custom attributes" and "actual attributes" are merged, and can "overlap"); something that should be looked at

Reply:
nit "exec_id"